src/transform/use_indexes.rs

```diff
@@ -38,29 +38,21 @@ impl crate::Transform for FilterEqualLiteral {
     fn transform(
         &self,
         relation: &mut RelationExpr,
-        indexes: &HashMap<GlobalId, Vec<Vec<ScalarExpr>>>,
+        state: &mut TransformArgs,
     ) -> Result<(), crate::TransformError> {
-        self.transform(relation, indexes);
+        self.transform(relation, state);
         Ok(())
     }
 }
 
 impl FilterEqualLiteral {
-    pub fn transform(
-        &self,
-        relation: &mut RelationExpr,
-        indexes: &HashMap<GlobalId, Vec<Vec<ScalarExpr>>>,
-    ) {
+    pub fn transform(&self, relation: &mut RelationExpr, state: &mut TransformArgs) {
         relation.visit_mut(&mut |e| {
-            self.action(e, indexes);
+            self.action(e, state);
```

Comment:
Imo it would be great to have this be `&state.indexes` to clarify that it only depends on the indexes. I think that ends up as a smaller diff too.

Reply:
This file doesn't even get built, it seems. Is that intentional?